sphinx/builders/html/__init__.py

```diff
@@ -1035,8 +1035,20 @@ def hasdoc(name: str) -> bool:
             templatename = newtmpl
 
         # sort JS/CSS before rendering HTML
-        ctx['script_files'].sort(key=lambda js: js.priority)
-        ctx['css_files'].sort(key=lambda js: js.priority)
+        try:
```

Comment:
FYI Python 3 has this nice pattern for such cases:
```suggestion
        with contextlib.suppress(AttributeError):
```